model/tf_call.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019-2020 Alain Dargelas
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Universal Hardware Data Model (UHDM) "tf_call" formal description
+
+- class_def: tf_call
```

Comment:
This generates a check for `vpiTfCall` in vpi_visitor.cpp, but there is no such define in vpi_user.h. Was that the case for any other model? I don't see a suitable substitution, should it be added in some UHDM-specific header?

Reply:
fixed by #139